Modelica/Fluid/Fittings.mo

```diff
@@ -549,12 +550,15 @@ model AbruptAdaptor
                 diameter_a/max(diameter_a, diameter_b)))*60},{0,-max(0.1, min(1,
                 diameter_a/max(diameter_a, diameter_b)))*60}}),
             fillPattern=FillPattern.HorizontalCylinder,
-            fillColor={0,127,255}), Rectangle(
+            fillColor={0,127,255},
+            pattern=LinePattern.None),
+                                    Rectangle(
             extent=DynamicSelect({{0,60},{100,-60}}, {{0,max(0.1, min(1,
                 diameter_b/max(diameter_a, diameter_b)))*60},{100,-max(0.1, min(
                 1, diameter_b/max(diameter_a, diameter_b)))*60}}),
             fillPattern=FillPattern.HorizontalCylinder,
-            fillColor={0,127,255})}),
+            fillColor={0,127,255},
+            pattern=LinePattern.None)}),
```

Comment:
I prefer the old one (left), they look more clearly like cylinders, instead of having them float into each other weirdly:

<img width="1538" height="562" alt="Image" src="https://github.com/user-attachments/assets/61f1ac76-a110-4f4b-a3eb-1c05b64853fd" />